tests/test_transform.py

```diff
@@ -187,45 +169,31 @@ def test_search_space_transform_values_categorical(
 
 def test_search_space_transform_untransform_params() -> None:
```

Comment:
I will ask you to resolve conflicts after merging #3181. :pray:

Reply:
No problem, done!